File: solutions/001/solution.js
const multipleOf = (num, multiple) => num % multiple == 0;
const sumofMultiples = (limit) => {
  let sum = 0;
  for(let x = 0; x < limit; x++) {
    if(multipleOf(x, 3) || multipleOf(x, 5)) {
      sum += x;
    }
  }
  return sum;
}
console.log(sumofMultiples(1000));

File: solutions/001/solution.py
def multiple_of(num, multiple):
  return num % multiple == 0

def sum_of_multiples(limit):
  return sum(x for x in range(limit) if multiple_of(x, 3) or multiple_of(x, 5))

if __name__ == "__main__":
	print(sum_of_multiples(1000))